### PR TITLE
[Ubuntu] Change source of n tool to official

### PR DESCRIPTION
**Pull request by nikolai-frolov**
## Description
This PR corrects source of n tool according to [official documentation](https://github.com/tj/n) and adds gathering of n version to Software Report.

## Related issue: 
https://github.com/actions/virtual-environments-internal/issues/2890

## Test builds:
- [Ubuntu 1804](https://github.visualstudio.com/virtual-environments/_build/results?buildId=118902&view=results)
- [Ubuntu 2004](https://github.visualstudio.com/virtual-environments/_build/results?buildId=118901&view=results)

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by miketimofeev — DISMISSED** [review]

**Comment by mspaansen** [comment]
5fa10884651b861453b12cbdbfd2ff6d5335fd7a

**Review by MaksimZhukov — DISMISSED** [review]

**Review by miketimofeev — DISMISSED** [review]

File: images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1 [hunk 1]
```diff
@@ -170,6 +170,11 @@ function Get-NewmanVersion {
     return "Newman $(newman --version)"
 }
 
+Get-NVersion {
```
Comment (miketimofeev) [inline images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1:173]: you missed the function keyword here
```suggestion
function Get-NVersion {
```

**Review by miketimofeev — APPROVED** [review]

**Pull request merged**